### DISCUSSION TITLE
fa/manager/basic-concepts-and-troubleshooting/The-correct-format-of-admin-link/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#3935] [Content Help کمک به اصلاح محتوا]
# fa/manager/basic-concepts-and-troubleshooting/The-correct-format-of-admin-link/

The proxy manager platform.

https://hiddify.com/fa/manager/basic-concepts-and-troubleshooting/The-correct-format-of-admin-link/

**Comment by alizarei54** [comment]
سلام ،من از طریق پنل آپدیت کردم حالا دیگه نمیتونم وارد پنل بسم باید چکار کنم
